### PR TITLE
Bump compass chart to the latest version

### PR DESCRIPTION
**Pull request by aszecowka**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: resources/compass/values.yaml [hunk 1]
```diff
@@ -13,10 +13,13 @@ global:
       path: eu.gcr.io/kyma-project/incubator
     connector:
       dir:
-      version: "ed26afa1"
+      version: "PR-634"
```
Comment (crabtree) [inline resources/compass/values.yaml]: Please put here the final image tag

File: resources/compass/charts/cockpit/values.yaml [hunk 2]
```diff
@@ -3,7 +3,8 @@ images:
     path: eu.gcr.io/kyma-project
   ui:
     dir:
-    version: ab79f919
+    version: 56a4db12
```
Comment (dariadomagala) [inline resources/compass/charts/cockpit/values.yaml]: `ab79f919` is a newer commit than `56a4db12`, why did you change it back?

Comment (aszecowka) [reply resources/compass/charts/cockpit/values.yaml]: thanks for spotting that, this was a version used in https://github.com/kyma-incubator/compass/blob/master/chart/compass/charts/cockpit/values.yaml
I revert that change.

File: resources/compass/charts/connectivity-adapter/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1beta2
```
Comment (Szymongib) [inline resources/compass/charts/connectivity-adapter/templates/deployment.yaml]:
```suggestion
apiVersion: apps/v1
```
`apps/v1beta2` is deprecated and is not supported from Kubernetes 1.16 up.

Comment (aszecowka) [reply resources/compass/charts/connectivity-adapter/templates/deployment.yaml]: thanks @Szymongib, I will solve this in another PR

**Comment by ebensom** [comment]
Hi,
This change breaks deployments of director and migrator-job with Cloud SQL DB. May I ask if the cloud_sql_proxy code deletions were intentional in the corresponding helm charts?
